src/main/environment/admin_ci.properties

```diff
@@ -18,4 +18,6 @@ spring.redis.host=localhost
 spring.main.allow-bean-definition-overriding=true
 
 #ELK logging file name
-logging.file.name=@env.ADMIN_API_LOGGING_FILE_NAME@
\ No newline at end of file
+logging.file.name=@env.ADMIN_API_LOGGING_FILE_NAME@
+
+common-url=http://localhost:8083/

```

Comment:
For Ci, please use the placeholder with @@ here.
Check how common API Base URL is mentioned in other CI files.